site/content/guides/fips.md

```diff
@@ -47,8 +47,107 @@ The Contour [Dockerfile][8] uses a multistage build that performs compilation in
 In order to minimize the `projectcontour/contour` image footprint, the final output image only consists of a single layer, containing a lone file: the statically compiled `contour` binary.
 The standard Contour build uses the upstream `golang` image as a build base, however we will have to swap that out to build Contour with BoringCrypto.
 
-We can use the Google-provided Go implementation that has patches on top of standard Go to enable integrating BoringCrypto.
-This is available to us in the [`goboring/golang`][9] container image we can use as a build base.
+### Go with BoringCrypto
+
+#### Go version 1.19 and higher
+
+Starting with Go 1.19, you can simply add [`GOEXPERIMENT=boringcrypto`][18] to enable integrating BoringCrypto for standard Go.
+
+First of all we need to change the `Dockerfile`, to declare a new arg: 
```

Comment:
ah sorry for the confusion, but I intended us to actually make these changes to the Makefile/Dockerfile so users don't have to modify them themselves, could you move these changes there? should also simplify this section

Reply:
Oh I see, never mind :)